main.go

```diff
@@ -58,6 +59,7 @@ var (
 	leaderElectionRenewDeadline   time.Duration
 	leaderElectionRetryPeriod     time.Duration
 	watchNamespace                string
+	watchLabelValue               string
```

Comment:
might be I'm missing something, but I can't find where this flag gets passed to the reconcilers.

Reply:
This was on me - reviewed my code 3 times and missed the obvious :sweat: